erd.md

```diff
@@ -3,141 +3,624 @@ Django ER Diagram
 ---
 
 erDiagram
+HistoricalAppRole {
```

Comment:
@Sepehr-Sobhani I did a project wide search of `previous_year_attributable_emissions` field and found it in this erd diagram. I found the command to generate it and looks like it was very out of date. Just want to confirm that we want these `Historical` tables in here.

Reply:
I believe so :) `django-diagram` generates ERD diagrams for us so it would be too much work to delete historical records every time we update the diagram. Note that this diagram has not been updated for a long time, and we have a tech debt card to make it a part of the pre-commit hook. This way, we can always have an updated diagram. (https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=erd&pane=issue&itemId=49865804)

Reply:
Thanks for the quick response! Sounds good, I'll keep this updated as is and maybe we can bring that ticket in next sprint since it should be an easy one.